ops/index-chart

```diff
@@ -18,7 +18,7 @@ cr index  --token ${GITHUB_TOKEN} \
           --package-path ./deploy/ \
           --index-path index.yaml \
           --git-repo k3k \
-          -o rancher
+          -o galal-hussein
```

Comment:
is this correct?

Reply:
fixing, it was done for local testing purposes, reverting now to rancher